go/libkb/upak_loader.go

```diff
@@ -500,11 +500,12 @@ func (u *CachedUPAKLoader) LoadKeyV2(ctx context.Context, uid keybase1.UID, kid
 		ret, key = upak.FindKID(kid)
 		if key != nil {
 			u.G().VDL.CLogf(ctx, VLog0, "- found kid in UPAK: %v", *ret)
-			break
+			return ret, key, linkMap, nil
 		}
 		ret = nil
 	}
-	return ret, key, linkMap, nil
+
+	return ret, key, linkMap, NotFoundError{}
```

Comment:
can this just be `nil, nil, nil, NotFoundError{}` ? Always tricky to return data and an error, best to avoid it.